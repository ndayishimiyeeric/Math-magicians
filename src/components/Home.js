import React from 'react';
import styles from './styles/Home.module.css';

const Home = () => {
  const random = 'radom';
  return (
    <div className={styles.showcase}>
      <h3>Welcome to our page!</h3>
      <p className={styles.para}>
        Lorem ipsum dolor sit amet consectetur adipisicing elit.
        Explicabo, quidem? Doloribus doloremque id, magni unde quibusdam
        omnis provident debitis. Incidunt deserunt consequatur, numquam
        placeat asperiores id
        {' '}
        {random}
        {' '}
        ipsum? Labore atque velit error distinctio
        nam iusto voluptatem! At, hic. Temporibus consectetur, debitis eum
        vero, reiciendis corporis, sapiente iusto magnam ullam voluptates ea!
        ipsum? Labore atque velit error distinctio
        nam iusto voluptatem! At, hic. Temporibus consectetur, debitis eum
        vero, reiciendis corporis, sapiente iusto magnam ullam voluptates ea!
      </p>
      <span className={styles.para}>
        Lorem ipsum dolor sit amet consectetur adipisicing elit.
        Necessitatibus vero recusandae ratione magni corporis odio
        consequatur eos. Illum, ut perferendis.
        ipsum? Labore atque velit error distinctio
        nam iusto voluptatem! At, hic. Temporibus consectetur, debitis eum
        vero, reiciendis corporis, sapiente iusto magnam ullam voluptates ea!
      </span>
    </div>
  );
};

export default Home;
