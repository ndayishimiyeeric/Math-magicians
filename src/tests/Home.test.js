import React from 'react';
import TestRenderer from 'react-test-renderer';
import Home from '../components/Home';

test('renders correctly', () => {
  const tree = TestRenderer
    .create(<Home />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
