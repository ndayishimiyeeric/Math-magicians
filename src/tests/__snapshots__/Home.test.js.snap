// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="showcase"
>
  <h3
    className="welcome"
  >
    Welcome to our page!
  </h3>
  <p
    className="para"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Explicabo, quidem? Doloribus doloremque id, magni unde quibusdam omnis provident debitis. Incidunt deserunt consequatur, numquam placeat asperiores id
     
    radom
     
    ipsum? Labore atque velit error distinctio nam iusto voluptatem! At, hic. Temporibus consectetur, debitis eum vero, reiciendis corporis, sapiente iusto magnam ullam voluptates ea! ipsum? Labore atque velit error distinctio
  </p>
  <span
    className="para"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Necessitatibus vero recusandae ratione magni corporis odio consequatur eos. Illum, ut perferendis. ipsum? Labore atque velit error distinctio nam iusto voluptatem! At, hic. Temporibus consectetur, debitis eum vero, reiciendis corporis, sapiente iusto magnam ullam voluptates ea!
  </span>
  <div
    className="center"
  >
    <span
      className="desc"
    >
      LETS CONNECT
    </span>
    <ul
      className="social"
    >
      <li>
        <a
          className="socialLink"
          href="https://github.com/ndayishimiyeeric"
          htmlTarget="_blank"
        >
          <svg
            fill="none"
            height="1em"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="2"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"
            />
          </svg>
        </a>
      </li>
      <li>
        <a
          className="socialLink"
          href="https://linkedin.com/in/nderic"
        >
          <svg
            baseProfile="tiny"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            version="1.2"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M8 19h-3v-10h3v10zm11 0h-3v-5.342c0-1.392-.496-2.085-1.479-2.085-.779 0-1.273.388-1.521 1.165v6.262h-3s.04-9 0-10h2.368l.183 2h.062c.615-1 1.598-1.678 2.946-1.678 1.025 0 1.854.285 2.487 1.001.637.717.954 1.679.954 3.03v5.647z"
            />
            <ellipse
              cx="6.5"
              cy="6.5"
              rx="1.55"
              ry="1.5"
            />
          </svg>
        </a>
      </li>
      <li>
        <a
          className="socialLink"
          href="https://twitter.com/odaltongain"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M15 3.784a5.63 5.63 0 0 1-.65.803 6.058 6.058 0 0 1-.786.68 5.442 5.442 0 0 1 .014.377c0 .574-.061 1.141-.184 1.702a8.467 8.467 0 0 1-.534 1.627 8.444 8.444 0 0 1-1.264 2.04 7.768 7.768 0 0 1-1.72 1.521 7.835 7.835 0 0 1-2.095.95 8.524 8.524 0 0 1-2.379.329 8.178 8.178 0 0 1-2.293-.325A7.921 7.921 0 0 1 1 12.52a5.762 5.762 0 0 0 4.252-1.19 2.842 2.842 0 0 1-2.273-1.19 2.878 2.878 0 0 1-.407-.8c.091.014.181.026.27.035a2.797 2.797 0 0 0 1.022-.089 2.808 2.808 0 0 1-.926-.362 2.942 2.942 0 0 1-.728-.633 2.839 2.839 0 0 1-.65-1.822v-.033c.402.227.837.348 1.306.362a2.943 2.943 0 0 1-.936-1.04 2.955 2.955 0 0 1-.253-.649 2.945 2.945 0 0 1 .007-1.453c.063-.243.161-.474.294-.693.364.451.77.856 1.216 1.213a8.215 8.215 0 0 0 3.008 1.525 7.965 7.965 0 0 0 1.695.263 2.15 2.15 0 0 1-.058-.325 3.265 3.265 0 0 1-.017-.331c0-.397.075-.77.226-1.118a2.892 2.892 0 0 1 1.528-1.528 2.79 2.79 0 0 1 1.117-.225 2.846 2.846 0 0 1 2.099.909 5.7 5.7 0 0 0 1.818-.698 2.815 2.815 0 0 1-1.258 1.586A5.704 5.704 0 0 0 15 3.785z"
            />
          </svg>
        </a>
      </li>
    </ul>
  </div>
</div>
`;
